source/dataman/CLS/CLSDbUpgrade1Pt1.cpp

```diff
@@ -0,0 +1,128 @@
+#include "CLSDbUpgrade1Pt1.h"
+
+#include "util/AMErrorMonitor.h"
+
+CLSDbUpgrade1Pt1::CLSDbUpgrade1Pt1(CLSAppController::CLSBeamlineID targetFacilityId, const QString &databaseNameToUpgrade, QObject *parent)
+	: AMDbUpgrade(databaseNameToUpgrade, parent)
+{
+	targetFacilityId_ = targetFacilityId;
+}
+
+CLSDbUpgrade1Pt1::~CLSDbUpgrade1Pt1()
+{
+
+}
+
+QStringList CLSDbUpgrade1Pt1::upgradeFromTags() const
+{
+	return QStringList();
```

Comment:
An empty QStringList here means that the database should always be considered, regardless of the 'current' upgrade version, right?